### DISCUSSION TITLE
Recordings Storage Maintenance?

### DISCUSSION THREAD

**Discussion by java007md** [mrlt8/docker-wyze-bridge#767] [Q&A] [answered]
If you are recording streams, how are you performing maintenance on those recordings? I did not see any options within the wyze-bridge itself.

What I am going to look at is implementing a cron job to potentially remove files over a certain age (remove recordings older than 30 days), and/or look at available storage and remove older recordings if storage is >= 75%.

Thoughts or other suggestions?

Thanks.

**Comment by JA16122000** [comment, marked as answer]
I'm using also a cron job.
Working just fine!
Depending on the size of your storage, mine is removing older files then 2 weeks every hour.

**Reply by aerodb** [reply]
could anyone provide some direction on how to set up a chron job? this will be a new one for me.

**Reply by java007md** [reply]
Well, here is the approach I implemented from a linux machine.

(recordings are stored on /mnt/recordings)

Add a crontab entry (via crontab -e) with your desired parameters. Below is my entry to run the cleanup script every hour at 23 minutes past the top of the hour

`23 * * * * /home/username/scripts/recordings_mnt_cleanup.sh`


-----------

Below is the cleanup script with differing retentions for the main and substreams, and varying retention between the cameras.

```
$ cat recordings_mnt_cleanup.sh 

#!/usr/bin/bash

find /mnt/recordings/ -iwholename "*-cam-*/*.mp4" -mtime +10 | xargs rm

find /mnt/recordings/ -iwholename "*-cam-*-sub*/*.mp4" -mtime +2 | xargs rm

find /mnt/recordings/ -iwholename "*-cam-*-sub*/*.jpg" -mtime +2 | xargs rm
find /mnt/recordings/ -iwholename "*-cam-50161/*.jpg" -mtime +3 | xargs rm
find /mnt/recordings/ -iwholename "*-cam-50162/*.jpg" -mtime +3 | xargs rm
find /mnt/recordings/ -iwholename "*-cam-50163/*.jpg" -mtime +3 | xargs rm

```

There could very well be a better approach, but this seems to be working pretty well.
